tests/java/third_parties/elasticsearch/testdata/main.java

```diff
@@ -0,0 +1,59 @@
+// Use bearer:expected java_third_parties_elasticsearch to flag expected findings
+
+public class Foo {
+  public void bad(User user) {
+    // ...
+    ElasticsearchClient esClient = new ElasticsearchClient(transport);
+    // bearer:expected java_third_parties_elasticsearch
+    Customer customer = new Customer("cust-1", user.email, user.name);
+
+    esClient.indices().create(c -> c
+        .index("customers")
+    );
+    IndexResponse response = esClient.index(i -> i
+        .index("customers")
+        .id(customer.getSku())
+        .document(customer)
+    );
```

Comment:
```suggestion
    Customer customer = new Customer("cust-1", user.email, user.name);

    esClient.indices().create(c -> c
        .index("customers")
    );
    IndexResponse response = esClient.index(i -> i
        .index("customers")
        .id(customer.getSku())
         // bearer:expected java_third_parties_elasticsearch
        .document(customer)
    );
```

Reply:
I tried to do it here but I haven't been really successful 😅

Reply:
🙏